cmd/mtaBuild.go

```diff
@@ -189,7 +189,16 @@ func runMtaBuild(config mtaBuildOptions,
 	if len(config.Extensions) != 0 {
 		call = append(call, fmt.Sprintf("--extensions=%s", config.Extensions))
 	}
-	call = append(call, "--target", "./")
+	if len(config.Source) > 0 {
+		call = append(call, fmt.Sprintf("--source=%s", config.Source))
+	} else {
+		call = append(call, "--source", "./")
+	}
+	if len(config.Target) > 0 {
+		call = append(call, fmt.Sprintf("--target=%s", config.Target))
+	} else {
+		call = append(call, "--target", "./")
```

Comment:
Is this the default? Can this be omitted?

Reply:
I tried to simplify the default values `./` by setting them in the metadata file in the latest commit.

Reply:
No, what I meant is that this is the default of the tool that we can omit to passt the option at all.

Reply:
Correct, the target and source parameters are optional and will be set to `./` during the `mbt build` execution when not explicitly set.

Reply:
I changed the logic to only append both parameters in the `mbt build` command when custom values are set.